web/src/components/users/FirstUser.jsx

```diff
@@ -166,6 +172,26 @@ export default function FirstUser() {
     setFormValues({ ...formValues, [name]: value });
   };
 
+  const suggestUsernames = (fullName) => {
+    const cleanedName = fullName.replace(/[^\p{L}\p{N} ]/gu, "").toLowerCase();
```

Comment:
At least for me, that line deserves an explanation / comment :exploding_head:

Reply:
Also, I'm not sure if it pays off.

Reply:
> NP: At least for me, that line deserves an explanation / comment 🤯

Haha, indeed, this was weird for me as well.
This has almost identical effect as the regular expression `/[^a-zA-Z0-9 ]/g`, which cleans the string from all characters except alphanumeric and blank spaces.
However, when I just used this initially, non English characters (like š, ć, č, ...) would also get removed.

So, after googling for a bit, I found that since ECMAScript 2018 Unicode property escapes are available for regular expressions in JavaScript. So, the '\p{L}' matches any character from any language and '\p{N}' matches any numeric character.

> Also, I'm not sure if it pays off.

Of course, it is another thing should something like this be used at all. When I coded this, the idea was that users could enter some special characters in their full names, like ',', '.', ';', '-', '"', etc,... And I supposed that this isn't something that should be a part of the username.

Reply:
Thanks a lot for the explanation!

If we keep it in the code, definitely it deserves a comment above it.

However, I think that "johnq.diaz" is a valid username for "John Q. Díaz" where "johnqdíaz" is not. If I'm right, we can sanitize the name in another way or with other regular expression. Either way, a comment in the code is more than welcome for future developers. 

Keep going!